### PR TITLE
Support Django 4.0

### PR DESCRIPTION
**Pull request by submarcos**

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -16,7 +16,8 @@ python =
 deps =
     dj22: django>=2.2,<2.3
     dj31: django>=3.1,<3.2
-    dj32: django>=3.2,<3.3
+    dj32: django>=3.2,<4.0
```
Comment (hramezani) [inline tox.ini:19]: Let's keep it `django>=3.2,<3.3`
```suggestion
    dj32: django>=3.2,<3.3
```

File: CHANGES.rst [hunk 2]
```diff
@@ -3,7 +3,7 @@ Changes
 =======
 
 - Drop support for Django 3.0
-- Add support for Django 3.2
+- Add support for Django 3.2 and 4.0
```
Comment (hramezani) [inline CHANGES.rst:6]: Please add a new line for Django 4.0

Comment (submarcos) [reply CHANGES.rst:6]: already done

File: tox.ini (hunk 1 above)
Comment (submarcos) [reply tox.ini:19]: ok, even if according django versionning 3.3 will never exists

**Comment by submarcos** [comment]
@hramezani thank you for your review

**Review by hramezani — APPROVED** [review]
Thanks @submarcos

**Pull request merged**